src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -110,7 +110,7 @@
     private Double maxScore;
 
     @Column(name = "bonus_points")
-    private Double bonusPoints;
+    private Double bonusPoints = 0;
```

Comment:
```suggestion
    private Double bonusPoints = 0.0;
```
Auto-boxing from `int` to `Double` is not possible.